book.rb

```diff
@@ -0,0 +1,20 @@
+# Public: Book class used to create book object
+#
+# All methods are class methods and should be called on the Book class.
+class Book
+  attr_reader :rental
```

Comment:
- [ ] We need both   _Getters_ and _Setters_ for _@rentals_ to be present, therefore kindly ensure to mark _rentals_ with _attr_accessor_, like:
```ruby
  attr_accessor :rentals